draft/index.html

```diff
@@ -124,20 +121,506 @@
         };
     </script>
 </head>
-
 <body>
-<section id="abstract" data-include="contents/abstract.md"></section>
 
-<section id="introduction" class="informative" data-include="contents/introduction.md"></section>
+<section id="abstract">
+
+## Zusammenfassung
+
+Ein Metadatenprofil für die Beschreibung von OER mit Fokus auf den deutschsprachigen Raum. Es basiert hauptsächlich auf [[?schema.org]] bzw. Learning Resource Metadata Initiative [[?LRMI]] und nutzt dazu Teile von Simple Knowledge Organization System [[?SKOS]].
```

Comment:
Der Verweis auf LRMI von DC wird irgendwie nur hier gemacht, aber nicht in l227.